advocacy_docs/dev-guides/deploy/windows.mdx

```diff
@@ -0,0 +1,149 @@
+---
+title: Installing PostgreSQL for development and testing on Microsoft Windows
+navTitle: On Windows
+description: Install PostgreSQL on your local Windows machine for development purposes.
+product: postgresql
+iconName: logos/Windows
+---
+
+## Prerequesites
+
+- Windows 10 build 16299 or later
+- An account with Administrator rights
+
+## Installing
+
+For a development machine, we'll use [WinGet](https://learn.microsoft.com/en-us/windows/package-manager/winget/) to download and install PostgreSQL:
+
+```
+winget install PostgreSQL.PostgreSQL.16
+__OUTPUT__
+Found PostgreSQL 16 [PostgreSQL.PostgreSQL.16] Version 16.4-1
+This application is licensed to you by its owner.
+Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
+Downloading https://get.enterprisedb.com/postgresql/postgresql-16.4-1-windows-x64.exe
+  ██████████████████████████████   356 MB /  356 MB
+Successfully verified installer hash
+Starting package install...
+Successfully installed
+```
+
+...where `16` is the major version of PostgreSQL we wish to install. This will install PostgreSQL in unattended mode, putting it into the default location (`C:\Program Files\PostgreSQL\`) with the default password (`postgres`) and installing both StackBuilder and PGAdmin.
```

Comment:
maybe link to Postgres docs on unattended mode for those unfamiliar?

Reply:
Also, maybe links to StackBuilder and PGAdmin? 🤷

Reply:
Huh. I'm realizing we don't really document unattended mode anywhere obvious (we document it for EPAS, but not the PostgreSQL installer!). I'll see if I can clarify that...

Good thinking on the linkin'